apps/studio/src/server/modules/page/page.router.ts

```diff
@@ -40,26 +42,32 @@ export const pageRouter = router({
   updatePage: protectedProcedure
     .input(updatePageSchema)
     .query(async ({ input, ctx }) => {
-      const parentId = ''
-      const pageName = ''
+      // NOTE: Need to do validation like checking for existence of the page
+      // and whether the user has write-access to said page
```

Comment:
hmm, why put this comment here ah? 
This seems like code that belongs in the middleware?

Reply:
i'm open to extending the `protectedProcedure` and putting this there, wdyt

Reply:
done